appengine/flexible/Pubsub/Pubsub.Sample/Controllers/HomeController.cs

```diff
@@ -108,6 +108,7 @@ public class PubSubPayload : JsonWebSignature.Payload
         }
         /// <summary>
         /// Handle authenticated push request coming from pubsub.
+        /// See the full sample in https://github.com/cy-yun/dotnet-docs-samples/blob/main/appengine/flexible/Pubsub/Pubsub.Sample/Controllers/HomeController.cs

```

Comment:
This is still showing your fork.

```suggestion
        /// See the full sample in https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/main/appengine/flexible/Pubsub/Pubsub.Sample/Controllers/HomeController.cs

```